rosdep/base.yaml

```diff
@@ -1290,12 +1294,12 @@ libcap-dev:
   arch: [libcap]
   debian: [libcap-dev]
   fedora: [libcap-devel]
-  gentoo: [sys-libs/libcap]
+  gentoo: ['=sys-libs/libcap-2*']
```

Comment:
I think the rule here should be only to specify a version in Gentoo if it's explicitly specified for the Ubuntu version of the package. Otherwise I don't think we should specify.